aws/app/lambda/invoke_audit_logs.sh

```diff
@@ -22,15 +31,17 @@ while :; do
   echo "Raw SQS Messages: $RAW_MESSAGE"
 
   if [[ $RAW_MESSAGE == "null" ]]; then
-    echo "No messages to process"
+    echo "No messages to process"s
```

Comment:
Is the extra `s` after the string useful or just a typo?

Reply:
typo, good catch.  Thinking I hit Ctrl-S too slow.